src/components/tabs/utils.ts

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+export const passPropsToChildren = (
+  children,
+  props: Record<string, any>,
+  allowedChildNodeTypes: any[] = []
```

Comment:
Could we tighten up this type?

```suggestion
  allowedChildNodeTypes: ReactElement[] = []
```

Reply:
TS was throwing errors, using `JSXElementConstructor<any>[]` instead.
`allowedChildNodeTypes: JSXElementConstructor<any>[] = []`